src/dynamicprompts/wildcards/wildcard_manager.py

```diff
@@ -43,6 +45,18 @@ def __init__(
         elif self._path:
             self._root_map = {"": [self._path]}
 
+    def set_sort_wildcards(self, value: bool):
+        """
+        Set whether wildcard values should be sorted.
+        """
+        self._sort_wildcards = value
+
+    def set_dedup_wildcards(self, value: bool):
+        """
+        Set whether wildcard values should be deduplicated.
+        """
+        self._dedup_wildcards = value
```

Comment:
`def set_...` doesn't smell very Pythonic. Maybe these should just be kwargs on WildcardManager's constructor, and the properties don't need to be `_private` either?

Reply:
I should have marked this as draft, I don't really want to use setters - ideally I would prefer to use kwargs on the constructor but I'm nervous of another fiasco like we had earlier this week.

Re properties not being private, isn't that point behind setters and getters?

Reply:
I mean, not `**kwargs`, but just

```python
def __init__(
        self,
        path: Path | str | None = None,
        wildcard_wrap=default_parser_config.wildcard_wrap,
        *,
        root_map: RootMap | None = None,
        sort_wildcards: bool = True,
        deduplicate_wildcards: bool = True,
    ) -> None:
```
Since they're after the kwarg-only star, there's no chance of a downstream user accidentally passing them wrong, and the defaults would remain the current ones.

As for setters, sure, but then you'd do

```
@property
def sort_wildcards(self) -> bool:
    return self._sort_wildcards

@sort_wildcards.setter
def sort_wildcards(self, value) -> None:
    self._sort_wildcards = bool(value)
```

- but better yet just not mark these as private since they _are_ okay to fiddle with at runtime (unlike `_path` and `_cache` etc., for which the class holds some internal assumptions (e.g. `_path` and `_tree` and `_root_map` are tied together).